src/lib/common/PrivacyStatement.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+  import { ExternalLink, Modal } from "lib/common";
+
+  export let open: boolean;
+</script>
+
+<Modal title="Privacy Statement" bind:open>
+  <div class="govuk-prose">
+    <p>
+      Plan Your Active Travel Schemes stores no personal data. The only data that is stored is your sketches, they are stored in local browser storage, not in a central database.
```

Comment:
```suggestion
      Plan Your Active Travel Schemes only stores your sketches, which contain no personal data. These sketches are stored in your browser's local storage and not sent over the network. We collect server-side logs to track usage and comply with auditing and security requirements. This tool uses MapTiler, an external service, for hosting maps and location search; please refer also to their <ExternalLink href="https://www.maptiler.com/privacy-policy/">privacy policy</ExternalLink>. ```

Not sure about the wording of the server-side logs. Suggest we look for examples in other services.